shared/general/host-ssh.md

```diff
@@ -0,0 +1,15 @@
+### Accessing the host OS
+
+Starting with CLI release 6.12.0, it is possible to SSH into the host OS rather than the application container. This gives you access to logs and tools for services that operate outside the scope of your application, such as **NetworkManager**, **Docker**, and the supervisor.
+
+__Warning:__ Making changes to running services and network configurations carries the risk of losing access to your device. Before making changes to the host OS of a remote device, it is best to test locally. When in doubt, [reach out][forums] to us for guidance. 
```

Comment:
We should mention that with host OS updates don't maintain changes over updates. And can even break the updater.

Reply:
Very good point